package.json

```diff
@@ -8,7 +8,7 @@
     "aframe-event-set-component": "^5.0.0",
     "aframe-orbit-controls": "^1.3.0",
     "aframe-react": "^4.4.0",
-    "bootstrap": "^5.1.1",
+    "bootstrap": "4.6",
```

Comment:
Any reason why you're not using `react-bootstrap`?

Reply:
Oh, you are, you're just dropping this version down

Reply:
Yeah, `react-bootstrap` wasn't playing nice with `bootstrap 5`